simtools/schemas/model_parameters/save_pe_with_amplitude.schema.yml

```diff
@@ -0,0 +1,38 @@
+%YAML 1.2
+---
+title: Schema for save_pe_with_amplitude model parameter
+version: 0.1.0
+meta_schema: simpipe-schema
+meta_schema_url: https://raw.githubusercontent.com/gammasim/simtools/main/simtools/schemas/model_parameter_and_data_schema.metaschema.yml
+meta_schema_version: 0.1.0
+name: save_pe_with_amplitude
+description: |-
+  True if photo-electron data is to be stored with the resulting amplitudes (in mean-p.e. units) in addition to the arrival times.
+  Storing of photo-electron data should be activated separately (see save_photons (bit 1) and store_photoelectrons parameters)
+  but non-zero values of this parameter will also activate save_photons bit 1.
+  Storing p.e. in internal calibration events is controlled separately by save_calibration_pe.
+data:
+  - type: int
```

Comment:
Would `type: boolean` be more appropriate? We have done that with all (most?) other parameters with this type of range (I think Konrad uses this because bool was not available in his C version when he started sim_telarray).

Reply:
Changed to boolean